icu4j/main/common_tests/src/test/java/com/ibm/icu/dev/test/number/NumberRangeFormatterTest.java

```diff
@@ -823,6 +823,22 @@ public void testFieldPositions() {
         }
     }
 
+    @Test
+    public void locale() {
```

Comment:
The name of the test is `locale()`? That seems weird.  Am I missing context in which it makes more sense?

Reply:
In the file just above this one, the test is also named `locale()`